go/app/main.go

```diff
@@ -102,6 +148,26 @@ func getItems(c echo.Context) error {
 	return c.JSON(http.StatusOK, itemsData)
 }
 
+func getItemById(c echo.Context) error {
+	id, _ := strconv.Atoi(c.Param("id"))
+	file, err := os.Open("items.json")
+	if err != nil {
+		c.Logger().Errorf("Error opening file: %s", err)
+		res := Response{Message: "Error opening file"}
+		return c.JSON(http.StatusInternalServerError, res)
+	}
+	defer file.Close()
+
+	itemsData := Items{}
+	err = json.NewDecoder(file).Decode(&itemsData)
+	if err != nil {
+		log.Print("JSONファイルからの変換に失敗", err)
+		return err
+	}
+	return c.JSON(http.StatusOK, itemsData.Items[id-1])
```

Comment:
You will want to test if `id` is a valid index in `itemsData.Items`. Otherwise `go` will panic when the index is either negative or beyond the length of `itemsData.Items`.